packages/harmony/src/components/expandable-nav-item/ExpandableNavItem.tsx

```diff
@@ -72,14 +62,22 @@ export const ExpandableNavItem = ({
 }: ExpandableNavItemProps) => {
   const [isOpen, setIsOpen] = useState(defaultIsOpen)
   const [isHovered, setIsHovered] = useState(false)
+  const [isMainActive, setIsMainActive] = useState(false)
   const theme = useTheme()
 
   const [ref, bounds] = useMeasure({
     polyfill: ResizeObserver
   })
 
   const handleMouseEnter = () => setIsHovered(true)
-  const handleMouseLeave = () => setIsHovered(false)
+  const handleMouseLeave = () => {
+    setIsHovered(false)
+    setIsMainActive(false)
+  }
+
+  const handleMainMouseDown = () => setIsMainActive(true)
+  const handleMainMouseUp = () => setIsMainActive(false)
```

Comment:
not so fond of these names, almost learning towards just inlining these. sounds like we have multiple mice

Reply:
hm but this way we can useCallback them, which i will do

Reply:
wdyt

Reply:
gotcha, then maybe a rename `handleMainContainerMouse<value>`